spec/message_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'message'
+
+describe Message do
+
+  let(:message) { described_class.new }
+
+  describe '#send_text' do
+    it 'sends a text to the customer' do
+      allow(message).to receive(:send_text)
+    end
+  end
```

Comment:
I feel like you could pass in a double of `Twilio` here and do this in a more sophisticated way.